clusterctl/examples/google/generate-yaml.sh

```diff
@@ -184,3 +186,5 @@ cat $ADDON_TEMPLATE_FILE \
   | sed -e "s/\$CLUSTER_NAME/$CLUSTER_NAME/" \
   | sed "s/\$LOADBALANCER_SA_KEY/$LOADBALANCER_SA_KEY/" \
   > $ADDON_GENERATED_FILE
+
+echo -e "\nYour generated cluster name is ''${CLUSTER_NAME}'"
```

Comment:
I think there is an extra `'` before `${CLUSTER_NAME}`.

Reply:
There is, fixing and retesting.

Reply:
Fixed and re-ran, the last few lines looks like this now:

```
Your public key has been saved in out/machine-controller-key.pub.
The key fingerprint is:
SHA256:YwcmDSyt8VmNtNwVH0dJbT74qcUNNnwDjZ8wxz2tNTA clusterapi
The key's randomart image is:
+---[RSA 2048]----+
|     o...o  oEBoB|
|    o o+oo.. *.X*|
|     =.o* .  .O=+|
|    . oo .   .*=o|
|        S .  .o+=|
|       . o     =.|
|              o  |
|             .   |
|                 |
+----[SHA256]-----+

Your generated cluster name is 'kqkv6tulrgs'

```